spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -3450,24 +3450,28 @@ def _handle_goto_uri_event(self, pos):
             self._last_hover_pattern_text = pattern_text
             return False
 
-    def go_to_uri_from_cursor(self, uri):
+    def go_to_uri_from_cursor(self, uri, testing=False):

```

Comment:
II need this to avoid actually opening any programs which could make the test fail since the focus is lost.